### PR TITLE
Workflow icon is missing a tooltip #5152

### PR DESCRIPTION
**Pull request by ashklianko**

### CODE REVIEW

**Comment by ashklianko** [comment]
Will check tests first

**Review by alansemenov — CHANGES_REQUESTED** [review]
Instead of having comparisons like `this.status === WorkflowStateStatus.INVALID` all over the code, let's add static methods like these inside `WorkflowStateManager`:

```
static isReady(status: WorkflowStateStatus): boolean {
    return status === WorkflowStateStatus.READY;
}
```

File: modules/lib/src/main/resources/assets/js/app/wizard/WorkflowStateManager.ts [hunk 1]
```diff
@@ -46,62 +40,54 @@ export class WorkflowStateManager {
...
-        }
+        thumbnailUploader.setStatus(this.status);
+        thumbnailUploader.toggleClass('status-hidden', isStatusHidden);
 
-        if (toolbarIcon) {
-            this.toggleWorkflowStateClasses(toolbarIcon);
-        }
+        toolbarIcon?.toggleClass(WorkflowStateStatus.INVALID, this.status === WorkflowStateStatus.INVALID);
+        toolbarIcon?.toggleClass(WorkflowStateStatus.READY, this.status === WorkflowStateStatus.READY);
+        toolbarIcon?.toggleClass(WorkflowStateStatus.IN_PROGRESS, this.status === WorkflowStateStatus.IN_PROGRESS);
```
Comment (alansemenov) [inline modules/lib/src/main/resources/assets/js/app/wizard/WorkflowStateManager.ts]: `toolbarIcon?.toggleClass(WorkflowStateStatus.INVALID, WorkflowStateStatus.isInvalid(this.status));`

etc.

**Review by alansemenov — CHANGES_REQUESTED** [review]

**Comment by ashklianko** [comment]
Will work after https://github.com/enonic/app-contentstudio/pull/5383 merged

**Review by alansemenov — APPROVED** [review]

**Pull request merged**